book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- According to the project requirements, please implement both setter and getter methods for the rental instance variable 🙏